File: semester-I_C-language-program/Largest number.c
#include<stdio.h>
#include<conio.h>
main ()
{
	int n,i;
	printf("Enter how many numbers:  ");
	scanf("%d",&n);
	
	int arr;
	
	printf("Enter %d numbers\n ",n);
	for( i=0;i<n;i++)
	{
	scanf("%d",&arr[i]);		
	}
	int largest =arr[0];
	for(int i=1;i<n;i++){
		if(arr[i]>largest){
			largest =arr[i];
		}
	}
	printf("Largest number is: %d\n ",largest);
    return 0;
}

File: semester-II_JavaProgram/LargestNumber.java
import java.util.Scanner;

class LargestNumber {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // Read number of elements
        System.out.print("Enter how many numbers: ");
        int n = sc.nextInt();

        // Read first number as largest
        System.out.print("Enter number 1: ");
        int largest = sc.nextInt();

        // Read remaining numbers
        for (int i = 2; i <= n; i++) {
            System.out.print("Enter number " + i + ": ");
            int num = sc.nextInt();

            if (num > largest) {
                largest = num;
            }
        }

        // Display result
        System.out.println("Largest number is: " + largest);

        sc.close();
    }
}